package.json

```diff
@@ -50,6 +50,15 @@
           "type": "boolean",
           "default": true,
           "description": "Controls whether dune tasks should be automatically detected."
+        },
+        "ocaml.dune.files": {
+          "scope": "resource",
+          "type": "array",
```

Comment:
let me know if instead of an array you would prefer an object like `{dune: false, dune-workspace: true, dune-project: true}`